_posts/en/newsletters/2021-04-28-newsletter.md

```diff
@@ -0,0 +1,183 @@
+    the interactive funding protocol that's included as part of
+    C-Lightning's experimental support for [dual funding][topic dual
+    funding].  FIXME:harding-to-update The draft was discussed during
+    Monday's Lightning Specification Meeting on IRC, where participants
+    FIXME:summary-of-reaction.
+
+- **Call for topics in layer-crossing workshop:** Antoine Riard
+  [posted][riard workshop] to both the Bitcoin-Dev and Lightning-Dev
+  mailing lists about an upcoming IRC-based workshop he plans to host to
+  discuss [challenges][riard zoology] with onchain transaction relay
+  that affect layer-two protocols such as LN.  The goal is to build
+  technical consensus among the participants about which proposals are
+  especially worth perusing so that developers and reviewers can focus
+  on those proposals in the short term.
+
+    The post proposes an agenda that includes [package relay][topic
+    package relay], fee sponsorship (see [Newsletter #116][news116
+    sponsorship]), moving from [BIP125][] opt-in Replace By Fee
+    ([RBF][topic rbf]) to full RBF, improving coordination of security
+    response between primarily onchain projects such as full nodes and
+    primarily offchain projects like LN nodes, and defining what mempool
+    and relay policies can be reasonably depended upon by layer two
+    protocols.  Riard also asks for additional topic suggestions from
+    anyone planning to attend, with May 7th being the deadline for
+    submissions.  The workshop will likely be held mid June.
+
+- **Support for ECDSA signature adaptors added to libsecp256k1-zkp:**
+  [signature adaptors][topic adaptor signatures] were originally
+  described for Bitcoin by Andrew Poelstra using [schnorr][topic
+  schnorr signatures]-based [multisignatures][topic multisignature].
+  This allows a single signature to do up to three things at once: (1)
+  prove its creator had access to a certain private key, (2) prove
+  knowledge of an encryption key pre-selected by another party, (3)
+  reveal a pre-selected encryption key to another party.  This allows a
+  signature alone to do many of the things we currently do with Bitcoin
+  scripts, suggesting adaptor signatures could be used as "scriptless
+  script".
+
+    Accomplishing the same on ECDSA is not as easy.  However, Lloyd
+    Fournier [suggested][fournier otves] it would be relatively simple
+    if we separated goal #1 (proof of private key) from goals #2 and #3
+    (proving and revealing encryption keys, AKA adaptors).  This
+    requires using one signature object as just a signature and another
+    signature object for the adaptors, so it uses `OP_CHECKMULTISIG` and
+    is not quite as scriptless as before.  It also requires a [security
+    warning][ecdh warning] related to reusing some of the involved keys
+    with Elliptic Curve Diffie Hellman (ECDH) key exchange and ElGamal
+    encryption   Beyond that, it makes signature adaptors entirely
```

Comment:
```suggestion
    encryption.  Beyond that, it makes signature adaptors entirely
```
Also, it may be clearer to replace "it" here with a specific subject.